hpOneView/resources/resource.py

```diff
@@ -281,12 +339,15 @@ def create(self, options, timeout=-1, custom_headers=None):
             Created resource.
         """
         logger.debug('Create (uri = %s, resource = %s)' %
-                     (self.URI, str(options)))
+                     (self.URI, str(data)))
 
-        resource = self.merge_default_values(options, self.DEFAULT_VALUES)
-
-        return self.__do_post(uri, resource, timeout, custom_headers)
+        resource = deepcopy(self.data) 
+        if data:
+            resource.update(self.data)
```

Comment:
just confused here between self.data and data, I think here it 'resource.update(data)' is required

Reply:
Looks like you are looking into older commits. The code is updated in the latest commit.